{"name": "# C5192 Settings button is functional", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5176_Add_a_new_device.py\", line 56, in dvc_is_on_the_pg\n    context.app.main_page.dvc_is_on_the_pg()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 1047, in dvc_is_on_the_pg\n    actual_text = wait.until(EC.visibility_of_element_located(IMEI_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1602552133614, "stop": 1602552136508}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1602552136508, "stop": 1602552139312}, {"name": "And Go to the Devices page https://devcloud.connectedio.com/devices", "status": "passed", "start": 1602552139313, "stop": 1602552139506}, {"name": "Then Click Add new device button on the right up corner", "status": "passed", "start": 1602552139506, "stop": 1602552139853}, {"name": "Then Fill required Device IMEI field 356961071557824", "status": "passed", "start": 1602552139854, "stop": 1602552140046}, {"name": "And Fill Device Name field if any TVM Test123 jjnkjn", "status": "passed", "start": 1602552140046, "stop": 1602552140211}, {"name": "Then Fill required Serial Number field 1806208A0279", "status": "passed", "start": 1602552140212, "stop": 1602552140335}, {"name": "And Click Save and refresh the page", "status": "passed", "start": 1602552140335, "stop": 1602552141582}, {"name": "Then Success is here - device is on the page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5176_Add_a_new_device.py\", line 56, in dvc_is_on_the_pg\n    context.app.main_page.dvc_is_on_the_pg()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 1047, in dvc_is_on_the_pg\n    actual_text = wait.until(EC.visibility_of_element_located(IMEI_HR)).text\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1602552141584, "stop": 1602552159771}, {"name": "Then Choose any device with IMEI number which is highlighted in blue active IMEI number and click on an active IMEI number", "status": "skipped", "start": 1602552161874, "stop": 1602552161874}, {"name": "Then Click on Settings button in right up corner and verify it has Reboot item in the drop-down menu", "status": "skipped", "start": 1602552161874, "stop": 1602552161874}], "start": 1602552128263, "stop": 1602552161874, "uuid": "ad3c24bf-3865-45ef-8f66-8bbfcdae9bbd", "historyId": "d4abe5d2a6daea82f958426f5790cda4", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5192 Settings button is functional"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}